src/fields/config/schema.ts

```diff
@@ -318,6 +326,7 @@ export const blocks = baseField.keys({
 export const richText = baseField.keys({
   type: joi.string().valid('richText').required(),
   name: joi.string().required(),
+  select: joi.func(),
```

Comment:
This should allow array aswell for a simple usecase.
() => { return ['title'] } works and is basically the same

Reply:
That is true, but I think it should be encouraged to use a function and return true as default

```ts
({collection}) => {
  if (isMyCollection(collection) return ['title']
  return true
}
```

Perhaps it should be better documented that this is the desired approach as relationships can be added to the richText without the richText knowing of it.